source/Core/BSP/Pine64/Vendor/SoC/gd32vf103/Board/pinecil/Source/GCC/gcc_gd32vf103_flashxip.ld

```diff
@@ -50,7 +50,7 @@ __ILM_RAM_SIZE = 0x00010000;
  * </h>
 */
 __RAM_BASE = 0x20000000;
-__RAM_SIZE = 0x00005000;
```

Comment:
Ah this could be fun..
The very first batch had the smaller ic fitted.
I _believe_ this never went out to public orders but ill touch base to confirm this.